dataactbroker/handlers/jobHandler.py

```diff
@@ -24,6 +24,10 @@ def getSubmissionById(self,submissionId):
         result = self.runUniqueQuery(query,"No submission with that ID","Multiple submissions with that ID")
         return result
 
+    def getSubmissionsByUserAgency(self,user,limit=5):
```

Comment:
I don't feel strongly either way....just wanted to check-in and see if there's a preferred method. I think your anticipation of a future pagination story makes sense, since the submission volume will pile up over time.